pcweb/pages/docs/components/overview.py

```diff
@@ -88,10 +90,15 @@ def components_overview():
             ),
         ),
         rx.tablet_and_desktop(
-            rx.hstack(
-                *definitions,
+            rx.grid(
```

Comment:
When I changed the definitions, the two boxes were no longer symmetrically sized in the hstack.... Using grid to make the equal split happen (not sure if it's the best way!)

Reply:
This way is fine!